### PR TITLE
import Cactus docs

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: blackbox/docs/getting_started/clients/erlang.txt [hunk 1]
```diff
@@ -0,0 +1,61 @@
+===============================
+Connecting to Crate with Erlang
+===============================
+
+The Crate Erlang client is built using
+`rebar3 <https://github.com/rebar/rebar3>`__ and we recommend you use it
```
Comment (chaudum) [inline blackbox/docs/getting_started/clients/erlang.txt]: double underscore?

File: blackbox/docs/getting_started/clients/erlang.txt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+nodes become unavailable.
+
+It's possible to create multiple clients on one erlang node, these are
+registered using a client spec tuple, such as ``{local, process_name}``.
+The process name must be unique on each node.
+
+Want to Know More?
+------------------
+
+-  More detailed documentation on the Crate Python client is available
```
Comment (chaudum) [inline blackbox/docs/getting_started/clients/erlang.txt]: Python? -> Erlang

File: blackbox/docs/getting_started/clients/index.txt [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+GoLang
+~~~~~~
+
+herenow wrote `a Golang driver for
+Crate <https://github.com/herenow/go-crate>`__ that implements the Go
+sql interface package.
+
+.. raw:: html
+
+   </p>
```
Comment (chaudum) [inline blackbox/docs/getting_started/clients/index.txt]: ????

File: blackbox/docs/getting_started/clients/java.txt [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+=============================
+
+Java Client
+-----------
+
+Deprecation Notice
+~~~~~~~~~~~~~~~~~~
+
+The Java client is deprecated in versions of Crate 0.57.0 and will be
+removed by 0.58.0 and later. Instead the recommended way to access Crate
```
Comment (chaudum) [inline blackbox/docs/getting_started/clients/java.txt]: The Java client was deprecated in versions of Crate **0.57.0** and is remove in **0.58** ..

Comment (chaudum) [inline blackbox/docs/getting_started/clients/java.txt]: IMO we should remove this page completely already

File: blackbox/docs/getting_started/clients/java.txt [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+Connecting to Crate with Java
+=============================
+
+Java Client
+-----------
+
+Deprecation Notice
+~~~~~~~~~~~~~~~~~~
+
+The Java client is deprecated in versions of Crate 0.57.0 and will be
```
Comment (chaudum) [inline blackbox/docs/getting_started/clients/java.txt]: make it even WARNING

File: blackbox/docs/getting_started/clients/php.txt [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+`DBAL <http://www.doctrine-project.org/projects/dbal.html>`__ data
+abstraction layers.
+
+PDO
+---
+
+Install Driver
+~~~~~~~~~~~~~~
+
+Install the driver by adding a dependency to your *composer.json* file:
```
Comment (chaudum) [inline blackbox/docs/getting_started/clients/php.txt]: \`\`composer.json\`\`

File: blackbox/docs/getting_started/connect/crash.txt [hunk 7]
```diff
@@ -0,0 +1,36 @@
+==================
+Crash, Crate's CLI
```
Comment (chaudum) [inline blackbox/docs/getting_started/connect/crash.txt]: \=============
CraSh (Crate Shell)
\=============

File: blackbox/docs/tools/squirrel.txt [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+5. Connect your alias to your crate instance.
+
+Usage
+-----
+
+You can browse Crate using the Object Tree Worksheet, that is the usual
+view you use for browsing your tables and get information about their
+schema.
+
+[browsing the content and schema of a table using squirrel]({% media
```
Comment (chaudum) [inline blackbox/docs/tools/squirrel.txt]: here's still some markdown and django template

File: blackbox/docs/tools/streamsets.txt [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+   However, some of the selected fields should be processed and
+   converted into float values. Therefore, we add an additional
+   processing `Field
+   Converter <https://streamsets.com/documentation/datacollector/latest/help/#Processors/FieldConverter.html#concept_is3_zkp_wq>`__
+   stage into the pipeline. [Convert string input values]({% media
+   '/media/1605/2-streamsets-dst-field-conversion.png' %})
+
+3. Create the taxi table in Crate.
+
+``sql  CREATE TABLE taxi (      hack_license STRING,      medallion STRING,      total_amount FLOAT,      tip_amount FLOAT  );``
```
Comment (chaudum) [inline blackbox/docs/tools/streamsets.txt]: please add linebreaks

File: blackbox/docs/tools/streamsets.txt [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+   JDBC destination configuration. In the JDBC tab of the Crate JDBC
+   origin we set the connection string (1), the SQL query for fetching
+   the data from the database (2), the offset value and column (3). For
+   more detail information on how to set the offset value and column see
+   the StreamSets
+   `documentation <https://streamsets.com/documentation/datacollector/1.1.3/help/#Origins/JDBCConsumer.html#concept_nxz_2kz_bs>`__.
+   [Configure the Crate JDBC driver]({% media
+   '/media/1605/3-streamsets-source-jdbc.png' %})
+
+**IMPORTANT**
```
Comment (chaudum) [inline blackbox/docs/tools/streamsets.txt]:
```
.. note::

    ...
```

File: blackbox/docs/getting_started/scale/kubernetes.txt [hunk 11]
```diff
@@ -0,0 +1,213 @@
+==========
+Kubernetes
```
Comment (chaudum) [inline blackbox/docs/getting_started/scale/kubernetes.txt]: this is probably out of date!

File: blackbox/docs/getting_started/scale/docker_swarm.txt [hunk 12]
```diff
@@ -0,0 +1,220 @@
+============
+Docker Swarm
```
Comment (chaudum) [inline blackbox/docs/getting_started/scale/docker_swarm.txt]: this is probably out of date

File: blackbox/docs/getting_started/install/local/tarball.txt [hunk 13]
```diff
@@ -0,0 +1,44 @@
+======================
+Install from a Tarball
+======================
+
+Latest Stable Release
+---------------------
+
+.. Version {{ CONFIG.stable\_version }}
```
Comment (chaudum) [inline blackbox/docs/getting_started/install/local/tarball.txt]: how is the version parsed into the document?

**Comment by chaudum** [comment]
all sections needs an overhaul. since the docs are now bound to a version, they need to comply to that version.

**Comment by mfussenegger** [comment]
I may be a bit late here, but wouldn't it make more sense to still have this kind of documentation outside of the core crate project? Otherwise we won't be able to update client docs without also having to do a crate release.


I think a structure like the following would still allow individual doc management and yet also have a "integrated" experience.

```
crate-docs/
    crate/ 
    clients/
        crash/
         ... 
    tools/
```

The crate folder would be a git submodule pointed to this repo. crate-docs would be it's own repo with it's own sphinx setup and it's own index.

In order to cross-link from one project to another we could use [intersphinx](http://www.sphinx-doc.org/en/stable/tutorial.html#intersphinx)

**Pull request closed without merging**